### PR TITLE
Added doc for Atome sensor

### PR DESCRIPTION
**Pull request by BaQs**
**Description:**

The `atome` sensor platform is retrieving the consumption of your home from the [Direct Energy Atome electric meter](https://total.direct-energie.com/particuliers/electricite/compteur-linky/atome).
This special little device is connected to a Linky Electric Meter, and sends live data to a cloud platform.

**Pull request in home-assistant (if applicable):**
https://github.com/home-assistant/home-assistant/pull/26197

## Checklist:

- [x ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10172"></a>

### CODE REVIEW

File: source/_components/atome.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: "Atome Linky Sensor"
+description: "Instructions on how to integrate Atome Linky consumption data within Home Assistant."
+logo: enedis.png
+ha_release: 0.97
+ha_category:
+  - Energy
+  - Sensor
+ha_iot_class: Cloud Polling
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/atome.markdown]: Please remove the `redirect_from`, this is only for existing pages

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/atome.markdown (hunk 1 above)
Comment (BaQs) [reply source/_components/atome.markdown]: Thx for reviewing, doing

File: source/_components/atome.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+title: "Atome Linky Sensor"
+description: "Instructions on how to integrate Atome Linky consumption data within Home Assistant."
+logo: enedis.png
+ha_release: 0.97
```
Comment (MartinHjelmare) [inline source/_components/atome.markdown]: 0.99

File: source/_components/atome.markdown [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---
+title: "Atome Linky Sensor"
+description: "Instructions on how to integrate Atome Linky consumption data within Home Assistant."
+logo: enedis.png
```
Comment (MartinHjelmare) [inline source/_components/atome.markdown]: Is this correct?

File: source/_components/atome.markdown [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+The `atome` sensor platform is retrieving the consumption of your home from the [Direct Energy Atome electric meter](https://total.direct-energie.com/particuliers/electricite/compteur-linky/atome).
+This special little device is connected to a Linky Electric Meter, and sends live data to a cloud platform.
+
+As there is no official documentation for the API, the component retrieves data from the API used in the Atome mobile app, [hosted here](http://esoftlink.esoftthings.com).
+
+## Configuration
+
+To use it, you need to order the device directly from "Total Direct Energie" Mobile App. Then you need to follow up the installation (covered inthe Atome App).
```
Comment (MartinHjelmare) [inline source/_components/atome.markdown]: covered in the Atome app.

File: source/_components/atome.markdown [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+{% configuration %}
+username:
+  description: The Atome account username.
+  required: true
+  type: string
+password:
+  description: The Atome account password.
+  required: true
+  type: string
+timeout:
```
Comment (MartinHjelmare) [inline source/_components/atome.markdown]: We removed this option I think.

File: source/_components/atome.markdown (hunk 2 above)
Comment (BaQs) [reply source/_components/atome.markdown]: indeed, solved

**Review by MartinHjelmare — DISMISSED** [review]

**Review by frenck — APPROVED** [review]
Thanks for the PR's @BaQs! Time to merge this in! 👍

**Pull request merged**